pages/onboarding/index.tsx

```diff
@@ -15,8 +67,77 @@ const Onboarding: NextPage = () => {
           </Button>
         </Link>
       </Box>
+
+      <form onSubmit={handleSubmit}>
+        <TextField
+          label="Bio"
+          name="bio"
+          value={formData.bio}
+          onChange={handleChange}
+          multiline
+          rows={4}
+          fullWidth
+          inputProps={{ maxLength: 1000 }}
+          required
+        />
+        <TextField
+          label="Twitter username"
+          name="twitter_username"
+          value={formData.twitter_username}
+          onChange={handleChange}
+          fullWidth
+          inputProps={{ maxLength: 200 }}
+          required
+        />
+        <TextField
+          label="LinkedIn URL"
+          name="linkedin_url"
+          value={formData.linkedin_url}
+          onChange={handleChange}
+          fullWidth
+          inputProps={{ maxLength: 200 }}
+          required
+        />
+        <TextField
+          label="GitHub username"
+          name="github_username"
+          value={formData.github_username}
+          onChange={handleChange}
+          fullWidth
+          inputProps={{ maxLength: 200 }}
+          required
+        />
+        <TextField
+          label="Website"
+          name="website"
+          value={formData.website}
+          onChange={handleChange}
+          fullWidth
+          inputProps={{ maxLength: 200 }}
+          required
+        />
+
+        <Box mt={4} textAlign="right">
+          <Button variant="contained" color="primary" type="submit">
+            Submit
```

Comment:
this button should be disabled if there is an in-flight request - it will prevent them from double-clicking it accidentally.